bootstrap/src/Pharo30Bootstrap/PBClassLoader.class.st

```diff
@@ -207,9 +207,12 @@ PBClassLoader >> initializeClassPool: aClassName [
 
 	| definition class classPool classVariableBindings |
 	definition := systemDefinition classNamed: aClassName.
+	class := environment at: aClassName.
+
 	classVariableBindings := definition classVarNames collect: [ :each | | classVar |
 		classVar := (self classNamed: #ClassVariable) basicNew.
-		classVar instanceVariableAtIndex: 1 put: (each asLiteralInObjectSpace: builder objectSpace) ].
+		classVar instanceVariableAtIndex: 1 put: (each asLiteralInObjectSpace: builder objectSpace).
+		classVar instanceVariableAtIndex: 3 put: class "initialize owningClass" ].
```

Comment:
@guillep could you review this bootstrap change please? Tests shows that it's fine. But maybe it should be done differently. 
Anyway I do not think it prevents merging. I can address your suggestion (if any) in another PR